.github/workflows/prod-deploy.yml

```diff
@@ -89,7 +89,9 @@ jobs:
         uses: docker/build-push-action@v3.1.1
         with:
           context: .
-          platforms: linux/amd64,linux/arm64
+          platforms: linux/amd64,linux/arm64,linux/arm/v7
```

Comment:
It fails at installing dependencies in case of `linux/arm/v7`
```
 > [linux/arm/v7 builder 4/6] RUN npm ci:
#26 273.0 npm ERR! code 1
#26 273.0 npm ERR! path /src/node_modules/sharp
#26 273.0 npm ERR! command failed
#26 273.0 npm ERR! command sh -c (node install/libvips && node install/dll-copy && prebuild-install) || (node install/can-compile && node-gyp rebuild && node install/dll-copy)
#26 273.0 npm ERR! sharp: Downloading https://github.com/lovell/sharp-libvips/releases/download/v8.10.6/libvips-8.10.6-linuxmusl-armv7.tar.br
#26 273.0 npm ERR! sharp: Please see https://sharp.pixelplumbing.com/install for required dependencies
#26 273.0 npm ERR! sharp: Installation error: Prebuilt libvips 8.10.6 binaries are not yet available for linuxmusl-armv7
```